frontend/src/root.jsx

```diff
@@ -89,6 +88,12 @@ api
       });
   });
 
+api.get("/stakeholder").then((resp) => {
```

Comment:
Should this be refactored to match other fetches being done in #406 ?

Reply:
@wayangalihpratama 
it seems weird to me the way we fetch in this #406 PR ... is the nested fetching approach really necessary? I wonder if there is a clean  approach in js world for that ... also i suspect the different calls are run in a sync way (as a sequence of http calls) so the full load will take more time that it should if we take an async approach (spawning all http calls at the same time )...

Reply:
I agree that we should do it async, and try to setup the dependencies correctly! I'm just wary of having incomplete data in the FE with the current way we fetch data.